includes/html/api_functions.inc.php

```diff
@@ -1665,6 +1665,7 @@ function list_oxidized(Illuminate\Http\Request $request)
     foreach ($devices as $device) {
         $output = [
             'hostname' => $device->hostname,
+            'sysname' => $device->sysName,
```

Comment:
Please update the docs to show this new output in the examples.

Reply:
What does adding this field do? I thought oxidized didn't accept any other fields.

Reply:
You can map the fields so if the sysname is more useful as the hostname then you can get oxidized to use that instead.

Reply:
I didn't think oxidized would accept random fields.

Reply:
You map the random field from LibreNMS to a known field in Oxidized